packages/snap-preact-components/src/components/Molecules/RadioSelect/RadioSelect.tsx

```diff
@@ -0,0 +1,174 @@
+/** @jsx jsx */
+import { Fragment, h } from 'preact';
+
+import { jsx, css } from '@emotion/react';
+import classnames from 'classnames';
+
+import { Theme, useTheme, CacheProvider } from '../../../providers';
+import { ComponentProps, StylingCSS } from '../../../types';
+import { defined, mergeProps } from '../../../utilities';
+import { Icon, IconProps } from '../../Atoms/Icon';
+import { useState } from 'react';
+
+const CSS = {
+	radioSelect: ({}: Partial<RadioSelectProps>) =>
+		css({
+			'& .ss__radio-select__options-wrapper': {
+				border: 'none',
+			},
+
+			'.ss__radio-select__title': {
+				margin: '0px',
+				padding: '5px',
+			},
+
+			'.ss__radio-select__option': {
+				cursor: 'pointer',
+				'& label': {
+					cursor: 'pointer',
+				},
+			},
+
+			'.ss__radio-select--selected': {
+				fontWeight: 'bold',
+			},
+
+			ul: {
+				listStyle: 'none',
+				padding: '0px',
+				margin: '0px',
+			},
+
+			li: {
+				display: 'flex',
+				alignItems: 'center',
+				padding: '5px',
+			},
+			'li label': {
+				padding: '0px 0px 0px 5px',
+			},
+		}),
+};
+
+export function RadioSelect(properties: RadioSelectProps): JSX.Element {
+	const globalTheme: Theme = useTheme();
+	const defaultProps: Partial<RadioSelectProps> = {};
+
+	const props = mergeProps('radioSelect', globalTheme, defaultProps, properties);
+
+	const { titleText, onSelect, hideRadios, native, disabled, selected, options, disableStyles, className, style, styleScript } = props;
+
+	const subProps: RadioSelectSubProps = {
+		CheckedIcon: {
+			// default props
+			width: 20,
+			height: 20,
+			viewBox: undefined,
+			children: <circle cx="10" cy="10" r="5" stroke="black" stroke-width="3" fill="gray"></circle>,
+			className: 'ss__list__option__radio__icon--checked',
+			// inherited props
+			...defined({
+				disableStyles,
+			}),
+			// component theme overrides
+			theme: props?.theme,
+		},
+		UncheckedIcon: {
+			// default props
+			width: 20,
+			height: 20,
+			viewBox: undefined,
+			children: <circle cx="10" cy="10" r="5" stroke="black" stroke-width="3" fill="white"></circle>,
+			className: 'ss__list__option__radio__icon--unchecked',
+			// inherited props
+			...defined({
+				disableStyles,
+			}),
+			// component theme overrides
+			theme: props?.theme,
```

Comment:
Do you think we ought to make some radio icons for the library instead of doing it this way?